### ISSUE TITLE
Deletion doesn't work

### ISSUE THREAD

**Issue by fluxa** [hfehrmann/reign_application#1]
Deletion does not seem to work very well in the app, when an item is deleted, items below it don't seem to change/move up.

**Comment by hfehrmann** [comment]
True. The problem was due to the persistance of the articles between deletion. I pushed a fix that persists the current articles between deletion and now the articles go up.

**Comment by fluxa** [comment]
Thank you @hfehrmann !
